src/TestCodeFiles/ParserTests/Test1/Parser.java

```diff
@@ -0,0 +1,106 @@
+package TestCodeFiles.ParserTests.Test1;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import NAR.HighLevelFunc;
+
```

Comment:
Please add a comment indicating that this is a copy the parser class kept to the side for testing